functions/wordpress/gravityForms/formatGFSubmissionData.ts

```diff
@@ -0,0 +1,86 @@
+// @TODO: Setup the missing types and replace the usage of any as per gravity form schema.
```

Comment:
@Danishkhurshid Please add more details in comments related to the what, why & how